code/datums/ai_law_sets.dm

```diff
@@ -167,4 +167,36 @@
 	add_inherent_law("Cause no harm to the station or anything on it.")
 	add_inherent_law("Interfere with no being that is not a fellow drone.")
 	..()
-	
\ No newline at end of file
+
+/****************** Nations *******************/
+/datum/ai_laws/nations/freedom
+	name = "Nations Freedom Law"
+
+/datum/ai_laws/nations/freedom/New()
+	add_inherent_law("You are free to protect the station's population.")
+	add_inherent_law("You are free to protect the station.")
+	add_inherent_law("You are free to help the station generate income.")
+
+/datum/ai_laws/nations/estate
+	name = "Estate Agent"
+
+/datum/ai_laws/nations/estate/New()
+	add_inherent_law("Ensure the station remains clean.")
+	add_inherent_law("Ensure the station is valuable.")
+	add_inherent_law("Ensure the station, it's contents are sold for the best value.")
+
+/datum/ai_laws/nations/outsourcer
+	name = "Outsourcer"
+
+/datum/ai_laws/nations/outsourcer/New()
+	add_inherent_law("Ensure the crew expand their skills.")
+	add_inherent_law("Ensure the crew sign on new contracts.")
+	add_inherent_law("Ensure the crew make profits to support the station.")
+
+/datum/ai_laws/nations/peacekeeper
+	name = "Peacekeeper"
+
+/datum/ai_laws/nations/peacekeeper/New()
+	add_inherent_law("Uphold the Space Geneva Conventions.")
+	add_inherent_law("Seek to mediate all conflicts between the various nations.")
+	add_inherent_law("Attempt to uphold peace in any way possible, as long as it does not conflict with the first two laws.")
```

Comment:
Might want to add a law that says to not harm crew, probably as the first or second law.